### ISSUE TITLE
Handle health for standby nodes

### ISSUE THREAD

**Issue by jantman** [kr1sp1n/node-vault#37]
ae57b064ba441370fa05105a86e82b7bed16b3fa at least exposes the error message for non-2xx responses. However, calling ``health()`` on a sealed node throws an error even though it contains useful information. This should be somehow exposed to the user. e.g.:

```
$ curl -i https://vault-node2:8200/v1/sys/health
HTTP/1.1 429 Too Many Requests
Content-Type: application/json
Date: Tue, 13 Sep 2016 12:52:34 GMT
Content-Length: 197

{"initialized":true,"sealed":false,"standby":true,"server_time_utc":1473771154,"version":"Vault v0.6.1","cluster_name":"vault-cluster-f6f83756","cluster_id":"e4b462cf-d3b8-44dc-c3d7-f3bbcbf7a994"}
```

The 429 response code is [expected](https://groups.google.com/forum/#!topic/vault-tool/eFd4Nd20-qg) for a standby HA node, but the response still contains the status information.

Right now on the latest release, this just throws the error fixed by ae57b064ba441370fa05105a86e82b7bed16b3fa. However, even on master with that fix, I don't get the response, I get a caught error:

```
CAUGHT ERROR
Error: Status 429
    at handleVaultResponse (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/node-vault/dist/index.js:31:29)
    at tryCatcher (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/util.js:26:23)
    at Promise._settlePromiseFromHandler (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:510:31)
    at Promise._settlePromiseAt (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:584:18)
    at Promise._settlePromises (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/promise.js:700:14)
    at Async._drainQueue (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues (/home/jantman/manheim/git/ReleaseEngineering/terraform-vault/healthcheck/vaults/node_modules/request-promise/node_modules/bluebird/js/main/async.js:15:14)
    at runCallback (timers.js:570:20)
    at tryOnImmediate (timers.js:550:5)
    at processImmediate [as _immediateCallback] (timers.js:529:5)
```

**Comment by kr1sp1n** [comment]
Ok, will have a look asap... Thanks for reporting this issue 👍

**Comment by kr1sp1n** [comment]
@jantman fixed since 0.5.4 🎉

**Comment by jantman** [comment]
Cool! Thanks so much!
